sdcm/nemesis.py

```diff
@@ -146,7 +146,7 @@ def _kill_scylla_daemon(self):
 
         # Let's wait for the target Node to have their services re-started
         self.log.info('Waiting scylla services to be restarted after we killed them...')
-        self.target_node.wait_db_up()
+        self.target_node.wait_db_up(timeout=14400)
```

Comment:
please add comment/explain in commit messsage why did you add this timeout

Reply:
I had explain in current commitlog, isn't it clear?

https://github.com/scylladb/scylla-cluster-tests/pull/704/commits/8f66ba2b18396d0383577767e84dee8dce6e90df

Reply:
@amoskong oh sorry, I didn't see the full commit message :+1:

Reply:
anyway, we need to find a way how to make this more robust, like querying Scylla. I will open the task for that in Trello. 
@roydahan Do we need to port this to `branch-3.0`?

Reply:
Yes, please.